libs/packages/components/src/lib/autocomplete-search/autocomplete-search.component.html

```diff
@@ -2,11 +2,12 @@
   <div class="position-relative maxw-mobile-lg">
     <div role="combobox" [attr.id]="configuration.id+'-container'" [attr.aria-expanded]="showResults"
       [attr.aria-owns]="showResults? configuration.id+ '-listbox' : undefined" aria-haspopup="listbox">
-      <input [disabled]="disabled" (ngModelChange)="textChange($event)" class="usa-input" #input
+      <input [disabled]="disabled" (input)="textChange($event)" class="usa-input" #input
```

Comment:
@thomastighe  Can you check this change may affect the formly autocomplete. Can you verify this change doesn't affect formly autocomplete?

Reply:
I did and it works